### DISCUSSION TITLE
docs/design/initialization

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#71] [General]
# docs/design/initialization

程序启动时需要执行的初始化操作，包括GoFrame框架中的隐式初始化和显式初始化。隐式初始化通过包的init方法实现，但可能导致程序启动失败，建议显式调用进行复杂初始化。显式初始化在业务开发中更为可取，以保证程序的可维护性。

https://goframe.org/docs/design/initialization

**Comment by juukee** [comment]
boot那几个文件可以参考一下吗？

**Reply by gqcn** [reply]
The images in the documentation are screenshots from a previous business project, and I no longer have the codes. Please refer to the screenshots instead.
